### ISSUE TITLE
Accessibility option ON - still not working

### ISSUE THREAD

**Issue by gdimo** [rxhanson/Middle-Community#70]
Hey guys,

I have purchased a license after the free trial but I have an issue. Even though the accessibility option is ON, the app still asks "enable Middle.app" and of course it's not working. Is there a way around that I am missing?

Thanks in advance :)

<img width="1066" alt="SCR-20240219-kncg" src="https://github.com/rxhanson/Middle-Community/assets/34108998/ca74b6e9-96ed-44f4-98b3-940e4f3ec683">

**Comment by rxhanson** [comment]
Looks like you figured out a solution? 

Typically when there are accessibility issues like this, it means the permissions are out of sync. 

For future reference, I've found that the easiest way to fix this is to use a terminal command to reset the accessibility permissions.

First, quit Middle.

`tccutil reset All com.knollsoft.Middle`

Then relaunch it.
